api/src/pcapi/routes/backoffice/templates/providers/list.html

```diff
@@ -1,58 +1,50 @@
+{% extends "layouts/list.html" %}
 {% import "components/links.html" as links with context %}
 {% from "components/turbo/lazy_modal.html" import build_lazy_modal with context %}
-{% extends "layouts/connected.html" %}
-{% block page %}
-  <div class="pt-3 px-5">
-    <h1>Synchronisation partenaires techniques</h1>
-    {% if has_permission("MANAGE_TECH_PARTNERS") %}
-      <button class="btn btn-outline-primary lead fw-bold mt-2"
-              data-bs-toggle="modal"
-              data-bs-target="#create-provider"
-              type="button">Créer un partenaire</button>
-      {{ build_lazy_modal(url_for("backoffice_web.providers.get_create_provider_form") , "create-provider") }}
-    {% endif %}
-    <div>
-      <table class="table mb-4">
-        <thead>
-          <tr>
-            <th scope="col"></th>
-            <th scope="col">ID</th>
-            <th scope="col">Partenaire technique</th>
-            <th scope="col">Partenaires culturels synchronisés au partenaire</th>
-            <th scope="col">SIREN</th>
-            <th scope="col">Ville</th>
-            <th scope="col">Code postal</th>
-            <th scope="col">Nombre de clés d'API</th>
-            <th scope="col">URL du logo</th>
-            <th scope="col">Actif pour les pros</th>
-            <th scope="col">Actif</th>
-          </tr>
-        </thead>
-        <tbody>
-          {% for row in rows %}
-            {% set provider = row.Provider %}
-            <tr>
-              <th scope="row"></th>
-              <td>{{ provider.id }}</td>
-              <td>{{ links.build_provider_name_to_details_link(provider) }}</td>
-              <td>{{ row.is_active_count | format_venue_provider_count }}</td>
-              <td>
-                {% if provider.offererProvider is not none %}{{ provider.offererProvider.offerer.siren }}{% endif %}
-              </td>
-              <td>
-                {% if provider.offererProvider is not none %}{{ provider.offererProvider.offerer.city }}{% endif %}
-              </td>
-              <td>
-                {% if provider.offererProvider is not none %}{{ provider.offererProvider.offerer.postalCode }}{% endif %}
-              </td>
-              <td>{{ provider.apiKeys | length }}</td>
-              <td>{{ provider.logoUrl | empty_string_if_null }}</td>
-              <td>{{ provider.enabledForPro | format_bool_badge }}</td>
-              <td>{{ provider.isActive | format_bool_badge }}</td>
-            </tr>
-          {% endfor %}
-        </tbody>
-      </table>
-    </div>
-  </div>
-{% endblock %}
+{% set rows_count = rows| length %}
+{% set hide_search_block = True %}
+{% block title %}Synchronisation partenaires techniques{% endblock %}
```

Comment:
Même si tu as gardé le titre existant, profitons-en pour bien s'exprimer :slightly_smiling_face: 
```suggestion
{% block title %}Synchronisation des partenaires techniques{% endblock %}
```

Reply:
1 oui, 2 oui, 3 si tu as une idée de quoi faire je suis preneur, je ne vois pas comment faire sans remettre le block en taille libre

Reply:
au moins la couleur des barres de défilement ? mais si c'est compliqué, laissons ainsi

Reply:
on peut mais je ne sais pas quelle couleur mettre donc je créé un ticket pour ça